docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md

```diff
@@ -6,19 +6,39 @@ description: Time tracking with Toggl
 keywords: time tracking, toggl integration
 ---
 
-# Time tracking with Toggl
+# Time tracking with Toggl Track
 
-To track time to a work package via a start and stop button, we offer an integration for the time recording software (or online timesheet software) Toggl (https://toggl.com/).
+To track time to a work package via a start and stop button, we offer an integration for the time recording software (or online timesheet software) Toggl Track ([https://toggl.com/track/](https://toggl.com/track/)).
```

Comment:
@adam-op I think it is worth it to explain/clarify that the spent time gets saved in Toggl Track and not in OpenProject.